test/controllers/findChannelTests.js

```diff
@@ -0,0 +1,15 @@
+const { expect } = require('chai');
+const Project = require('../../skills/findChannel.js');
+
+
+describe('Find Channel Tests', () => {
+  it('tests if channel exists', async () => {
+    console.log('1',Project)
```

Comment:
We should never leave console.logs in our production code like this. If the test assertions are not clear enough, let’s resolve that

Reply:
will do